jujugui/static/gui/src/app/components/budget-table/row/test-row.js

```diff
@@ -223,11 +223,11 @@ describe('BudgetTableRow', function() {
                 <div className="two-col">
                   Recommended allocation: $550.
                 </div>
-                <div className="two-col last-col">
+                <div className="two-col last-col v1">
                   <Button
                     action={wrapper.find('Button').at(1).prop('action')}
                     disabled={false}
-                    type="neutral">
+                    modifier="neutral">
```

Comment:
Some of the props have changed for the buttons as they've been updated to the new Vanilla.